controllers/imagerepository_controller.go

```diff
@@ -70,6 +65,7 @@ type ImageRepositoryReconciler struct {
 
 func (r *ImageRepositoryReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
+	reconcileStart := time.Now()
```

Comment:
Do you need this _and_ `now` below?

Reply:
Good point! It's probably just a few milliseconds apart, but I think this is the right value to use regardless of that.